### ISSUE TITLE
Plugin LivecodingTV fails to load on Python 3.6.0b2 on Windows 10 x64

### ISSUE THREAD

**Issue by plneappl** [streamlink/streamlink#106]
Just running streamlink raises the following error on my system with a fresh install:

```
C:\WINDOWS\system32>streamlink
Failed to load plugin livecodingtv:
  File "c:\program files\python36\lib\imp.py", line 234, in load_module
    return load_source(name, filename, file)
  File "c:\program files\python36\lib\imp.py", line 172, in load_source
    module = _load(spec)
  File "<frozen importlib._bootstrap>", line 675, in _load
  File "<frozen importlib._bootstrap>", line 655, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 677, in exec_module
  File "<frozen importlib._bootstrap>", line 205, in _call_with_frames_removed
  File "c:\program files\python36\lib\site-packages\streamlink\plugins\livecodingtv.py", line 9, in <module>
    _url_re = re.compile('http(s)?://(?:\w+.)?\livecoding\.tv')
  File "c:\program files\python36\lib\re.py", line 233, in compile
    return _compile(pattern, flags)
  File "c:\program files\python36\lib\re.py", line 301, in _compile
    p = sre_compile.compile(pattern, flags)
  File "c:\program files\python36\lib\sre_compile.py", line 562, in compile
    p = sre_parse.parse(p, flags)
  File "c:\program files\python36\lib\sre_parse.py", line 856, in parse
    p = _parse_sub(source, pattern, flags & SRE_FLAG_VERBOSE, False)
  File "c:\program files\python36\lib\sre_parse.py", line 415, in _parse_sub
    itemsappend(_parse(source, state, verbose))
  File "c:\program files\python36\lib\sre_parse.py", line 501, in _parse
    code = _escape(source, this, state)
  File "c:\program files\python36\lib\sre_parse.py", line 401, in _escape
    raise source.error("bad escape %s" % escape, len(escape))
sre_constants.error: bad escape \l at position 20

usage: streamlink [OPTIONS] [URL] [STREAM]

Use -h/--help to see the available options or read the manual at http://docs.streamlink.io/

C:\WINDOWS\system32>python --version
Python 3.6.0b2
```

**Comment by intact** [comment]
change https://github.com/streamlink/streamlink/blob/master/src/streamlink/plugins/livecodingtv.py#L9 to
```
_url_re = re.compile(r"http(s)?://(?:\w+\.)?livecoding\.tv")
```

**Comment by beardypig** [comment]
Submitted a PR to fix this: #121
